app/views/hub/verification_attempts/show.html.erb

```diff
@@ -19,18 +19,26 @@
         <%end %>
       </ul>
 
-      <div class="grid-flex">
-        <% if @verification_attempt.selfie.present? %>
-          <div class="item-15r">
-            <%= image_tag transient_storage_url(@verification_attempt.selfie.blob), id: "selfie" %>
+      <% if @verification_attempt.client_bypass_request.present? %>
+        <div class="note">
+          <div class="note__body">
+            <%= simple_format(@verification_attempt.client_bypass_request) %>
```

Comment:
trying to figure out what this looks like in the UI...is it just a note that says something like bypass?

Reply:
This will show whatever text the client submitted as to why they should have their request approved without providing photos. So in the case that they don't have an ID, they might say something like "I currently don't have access to my government issued ID."

"I actually am unhoused right now and I don't have access to a device with a camera"